observability/otel/otel-collector/otel-collector-connectors.adoc

```diff
@@ -8,6 +8,106 @@ toc::[]
 
 A connector connects two pipelines. It consumes data as an exporter at the end of one pipeline and emits data as a receiver at the start of another pipeline. It can consume and emit data of the same or different data type. It can generate and emit data to summarize the consumed data, or it can merely replicate or route data.
 
+[id="count-connector_{context}"]
+== Count Connector
+
+The Count Connector counts trace spans, trace span events, metrics, metric data points, and log records in exporter pipelines.
+
+:FeatureName: The Count Connector
+include::snippets/technology-preview.adoc[]
+
+The following are the default metric names:
+
+* `trace.span.count`
+* `trace.span.event.count`
+* `metric.count`
+* `metric.datapoint.count`
+* `log.record.count`
+
+You can also expose custom metric names.
+
+.OpenTelemetry Collector custom resource (CR) with an enabled Count Connector
+[source,yaml]
+----
+# ...
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc: # <1>
+            endpoint: 0.0.0.0:4317
+    exporters:
+      prometheus: # <2>
+        endpoint: 0.0.0.0:8889
+    connectors:
+      count:
+    service:
+      pipelines: # <3>
+        traces/in:
+          receivers: [otlp]
+          exporters: [count] # <4>
+        metrics/out:
+          receivers: [count] # <5>
+          exporters: [prometheus]
+# ...
+----
+<1> This example uses the OTLP Receiver.
+<2> This example uses the Prometheus Exporter.
+<3> It is important to correctly configure the Count Connector as an exporter or receiver in the pipeline and to export the generated metrics to the correct exporter.
+<4> In this example, the Count Connector is configured to receive spans as an exporter.
+<5> In this example, the Count Connector is configured to emit generated metrics as a receiver.
```

Comment:
> In <1>'s description, it's not clear if grpc is a name of the receiver or a type of the receiver or a protocol that communicates with the receiver. Can you please check with SME/QE to clarify?
`grpc` is the protocol used by the receiver

> Similar comment for <2>'s description. Can users use exporters other than Prometheus, are they supported?
Yes. They can use others. The original change was correct.

Reply:
Thank you, both.
I deleted the callouts `<1>` and `<2>` because they are self-explanatory.
I also shortened the callouts `<4>` and `<5>` even more than suggested.